### PR TITLE
[MODULAR] yeah yeah, blueshield time

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

Resprites some Blueshield stuff

New Armour Vest:
blueshieldarmour

New Turtleneck and Skirtleneck:
blueshieldturtle

Also improves some of the code for the blueshield clothing

## How This Contributes To The Skyrat Roleplay Experience

no

## Changelog

:cl: axietheaxolotl
imageadd: new blueshield drip
code: improves some of the code for blueshield clothing
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/blueshield/code/blueshield_clothing.dm [hunk 1]
```diff
@@ -1,8 +1,7 @@
-
 //Blueshield
 /obj/item/clothing/under/rank/security/blueshield
-	desc = "Expensive designer shirt with snazzy suit pants, complete with a blue tie."
-	name = "Blueshield's suit"
+	desc = "A expensive designer shirt with snazzy suit pants, complete with a blue tie."
```
Comment (tf-4) [inline modular_skyrat/modules/blueshield/code/blueshield_clothing.dm]:
```suggestion
	desc = "An expensive designer shirt with snazzy suit pants, complete with a blue tie."
```

File: modular_skyrat/modules/blueshield/code/blueshield_clothing.dm [hunk 2]
```diff
@@ -12,29 +11,22 @@
...
 
-/obj/item/clothing/under/rank/security/blueshieldturtleneck
-	name = "Blueshield's turtleneck"
-	desc = "A cozier alternative to the normal Blueshield's Suit, the wool is still expensive."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
-	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
+/obj/item/clothing/under/rank/security/blueshield/turtleneck
+	name = "blueshield's skivvy"
+	desc = "A cozier alternative to the normal Blueshield's Suit, made out of expensive all-natural wool."
```
Comment (tf-4) [inline modular_skyrat/modules/blueshield/code/blueshield_clothing.dm]:
```suggestion
	desc = "A cozier alternative to the normal Blueshield's suit. It's made out of an expensive, all-natural wool."
```

File: modular_skyrat/modules/blueshield/code/blueshield_clothing.dm [hunk 3]
```diff
@@ -12,29 +11,22 @@
...
-	can_adjust = FALSE
-	sensor_mode = SENSOR_COORDS
-	random_sensor = FALSE
+	fitted = FEMALE_UNIFORM_TOP
 
-/obj/item/clothing/under/rank/security/blueshieldskirt
-	desc = "A tactical skirt, seemingly outfitted in NanoTrasen's standard corporate-chic."
+/obj/item/clothing/under/rank/security/blueshield/turtleneck/skirt
+	name = "blueshield's skirtleneck"
+	desc = "A alternative to the normal turtleneck, with the pants replaced with a low-cut skirt. It's still made out of expensive all-natural wool."
```
Comment (tf-4) [inline modular_skyrat/modules/blueshield/code/blueshield_clothing.dm]:
```suggestion
	desc = "An alternative to the normal turtleneck with the pants replaced with a low-cut skirt. It's still made out of an expensive, all-natural wool."
```

File: modular_skyrat/modules/blueshield/code/blueshield_clothing.dm [hunk 4]
```diff
@@ -12,29 +11,22 @@
...
+	icon_state = "bs_skirtleneck"
+
+/obj/item/clothing/under/rank/security/blueshield/skirt
+	desc = "A \"tactical\" skirt, seemingly outfitted in Nanotrasen's standard corporate-chic."
```
Comment (tf-4) [inline modular_skyrat/modules/blueshield/code/blueshield_clothing.dm]:
```suggestion
	desc = "A \"tactical\" skirt seemingly outfitted in Nanotrasen's standard corporate-chic."
```

**Review by GoldenAlpharex — APPROVED** [review]
Looks alright.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**